Goblins.Core/GoblinHatchery.cs

```diff
@@ -1,23 +1,33 @@
-﻿namespace Goblins.Core
+﻿using Xunit.Abstractions;
```

Comment:
So... what does the Xunit.Abstractions do exactly? Is it needed for the ITestOutputHelper?